### PR TITLE
Fix test_tablet_missing_data_repair

### PR DESCRIPTION
**Pull request by asias**
This PR fixes test_tablet_missing_data_repair and enable the test again. 

If a node is not UP yet, repair in the test will be a partial repair. The partial repair will not repair all the data which cause the check of rows after repair to fail.  Check nodes see each other as UP before repair.

### CODE REVIEW

**Comment by asias** [comment]
Two 100 out of 100 passed in a row. 

Here is one of the logs. 

```
Found 100 tests.
================================================================================
[N/TOTAL]   SUITE    MODE   RESULT   TEST
------------------------------------------------------------------------------
[1/100]    topology_experimental_raft  debug  [ PASS ] topology_experimental_raft.test_tablets.1 67.95s
[2/100]    topology_experimental_raft  debug  [ PASS ] topology_experimental_raft.test_tablets.4 70.67s
[3/100]    topology_experimental_raft  debug  [ PASS ] topology_experimental_raft.test_tablets.2 72.21s
[4/100]    topology_experimental_raft  debug  [ PASS ] topology_experimental_raft.test_tablets.3 87.97s
[5/100]    topology_experimental_raft  debug  [ PASS ] topology_experimental_raft.test_tablets.5 73.18s
[6/100]    topology_experimental_raft  debug  [ PASS ] topology_experimental_raft.test_tablets.7 72.02s
[7/100]    topology_experimental_raft  debug  [ PASS ] topology_experimental_raft.test_tablets.6 90.36s
[8/100]    topology_experimental_raft  debug  [ PASS ] topology_experimental_raft.test_tablets.8 75.24s
[9/100]    topology_experimental_raft  debug  [ PASS ] topology_experimental_raft.test_tablets.10 71.76s
[10/100]   topology_experimental_raft  debug  [ PASS ] topology_experimental_raft.test_tablets.9 89.63s
[11/100]   topology_experimental_raft  debug  [ PASS ] topology_experimental_raft.test_tablets.12 70.73s
[12/100]   topology_experimental_raft  debug  [ PASS ] topology_experimental_raft.test_tablets.11 87.46s
[13/100]   topology_experimental_raft  debug  [ PASS ] topology_experimental_raft.test_tablets.13 71.43s
[14/100]   topology_experimental_raft  debug  [ PASS ] topology_experimental_raft.test_tablets.14 74.98s
[15/100]   topology_experimental_raft  debug  [ PASS ] topology_experimental_raft.test_tablets.16 71.03s
[16/100]   topology_experimental_raft  debug  [ PASS ] topology_experimental_raft.test_tablets.15 87.24s
[17/100]   topology_experimental_raft  debug  [ PASS ] topology_experimental_raft.test_tablets.17 74.22s
[18/100]   topology_experimental_raft  debug  [ PASS ] topology_experimental_raft.test_tablets.18 73.22s
[19/100]   topology_experimental_raft  debug  [ PASS ] topology_experimental_raft.test_tablets.19 72.16s
[20/100]   topology_experimental_raft  debug  [ PASS ] topology_experimental_raft.test_tablets.20 74.08s
[21/100]   topology_experimental_raft  debug  [ PASS ] topology_experimental_raft.test_tablets.21 73.07s
[22/100]   topology_experimental_raft  debug  [ PASS ] topology_experimental_raft.test_tablets.22 72.81s
[23/100]   topology_experimental_raft  debug  [ PASS ] topology_experimental_raft.test_tablets.23 73.69s
[24/100]   topology_experimental_raft  debug  [ PASS ] topology_experimental_raft.test_tablets.24 70.80s
[25/100]   topology_experimental_raft  debug  [ PASS ] topology_experimental_raft.test_tablets.25 73.86s
[26/100]   topology_experimental_raft  debug  [ PASS ] topology_experimental_raft.test_tablets.26 72.68s
[27/100]   topology_experimental_raft  debug  [ PASS ] topology_experimental_raft.test_tablets.27 71.48s
[28/100]   topology_experimental_raft  debug  [ PASS ] topology_experimental_raft.test_tablets.28 71.87s
[29/100]   topology_experimental_raft  debug  [ PASS ] topology_experimental_raft.test_tablets.29 71.09s
[30/100]   topology_experimental_raft  debug  [ PASS ] topology_experimental_raft.test_tablets.30 72.30s
[31/100]   topology_experimental_raft  debug  [ PASS ] topology_experimental_raft.test_tablets.31 73.62s
[32/100]   topology_experimental_raft  debug  [ PASS ] topology_experimental_raft.test_tablets.32 74.01s
[33/100]   topology_experimental_raft  debug  [ PASS ] topology_experimental_raft.test_tablets.33 73.12s
[34/100]   topology_experimental_raft  debug  [ PASS ] topology_experimental_raft.test_tablets.36 71.68s
[35/100]   topology_experimental_raft  debug  [ PASS ] topology_experimental_raft.test_tablets.34 87.46s
[36/100]   topology_experimental_raft  debug  [ PASS ] topology_experimental_raft.test_tablets.35 75.45s
[37/100]   topology_experimental_raft  debug  [ PASS ] topology_experimental_raft.test_tablets.37 72.24s
[38/100]   topology_experimental_raft  debug  [ PASS ] topology_experimental_raft.test_tablets.38 72.93s
[39/100]   topology_experimental_raft  debug  [ PASS ] topology_experimental_raft.test_tablets.39 73.27s
[40/100]   topology_experimental_raft  debug  [ PASS ] topology_experimental_raft.test_tablets.40 72.25s
[41/100]   topology_experimental_raft  debug  [ PASS ] topology_experimental_raft.test_tablets.41 87.53s
[42/100]   topology_experimental_raft  debug  [ PASS ] topology_experimental_raft.test_tablets.44 70.63s
[43/100]   topology_experimental_raft  debug  [ PASS ] topology_experimental_raft.test_tablets.42 88.64s
[44/100]   topology_experimental_raft  debug  [ PASS ] topology_experimental_raft.test_tablets.43 89.93s
[45/100]   topology_experimental_raft  debug  [ PASS ] topology_experimental_raft.test_tablets.46 71.33s
[46/100]   topology_experimental_raft  debug  [ PASS ] topology_experimental_raft.test_tablets.45 89.46s
[47/100]   topology_experimental_raft  debug  [ PASS ] topology_experimental_raft.test_tablets.48 70.43s
[48/100]   topology_experimental_raft  debug  [ PASS ] topology_experimental_raft.test_tablets.47 73.10s
[49/100]   topology_experimental_raft  debug  [ PASS ] topology_experimental_raft.test_tablets.49 72.27s
[50/100]   topology_experimental_raft  debug  [ PASS ] topology_experimental_raft.test_tablets.50 73.48s
[51/100]   topology_experimental_raft  debug  [ PASS ] topology_experimental_raft.test_tablets.52 72.82s
[52/100]   topology_experimental_raft  debug  [ PASS ] topology_experimental_raft.test_tablets.51 86.30s
[53/100]   topology_experimental_raft  debug  [ PASS ] topology_experimental_raft.test_tablets.53 74.12s
[54/100]   topology_experimental_raft  debug  [ PASS ] topology_experimental_raft.test_tablets.54 73.89s
[55/100]   topology_experimental_raft  debug  [ PASS ] topology_experimental_raft.test_tablets.55 71.43s
[56/100]   topology_experimental_raft  debug  [ PASS ] topology_experimental_raft.test_tablets.56 88.45s
[57/100]   topology_experimental_raft  debug  [ PASS ] topology_experimental_raft.test_tablets.57 71.83s
[58/100]   topology_experimental_raft  debug  [ PASS ] topology_experimental_raft.test_tablets.58 70.72s
[59/100]   topology_experimental_raft  debug  [ PASS ] topology_experimental_raft.test_tablets.59 73.11s
[60/100]   topology_experimental_raft  debug  [ PASS ] topology_experimental_raft.test_tablets.60 71.56s
[61/100]   topology_experimental_raft  debug  [ PASS ] topology_experimental_raft.test_tablets.62 73.24s
[62/100]   topology_experimental_raft  debug  [ PASS ] topology_experimental_raft.test_tablets.61 89.54s
[63/100]   topology_experimental_raft  debug  [ PASS ] topology_experimental_raft.test_tablets.63 88.70s
[64/100]   topology_experimental_raft  debug  [ PASS ] topology_experimental_raft.test_tablets.64 74.59s
[65/100]   topology_experimental_raft  debug  [ PASS ] topology_experimental_raft.test_tablets.65 73.35s
[66/100]   topology_experimental_raft  debug  [ PASS ] topology_experimental_raft.test_tablets.66 87.52s
[67/100]   topology_experimental_raft  debug  [ PASS ] topology_experimental_raft.test_tablets.67 72.36s
[68/100]   topology_experimental_raft  debug  [ PASS ] topology_experimental_raft.test_tablets.68 84.09s
[69/100]   topology_experimental_raft  debug  [ PASS ] topology_experimental_raft.test_tablets.69 72.69s
[70/100]   topology_experimental_raft  debug  [ PASS ] topology_experimental_raft.test_tablets.70 70.37s
[71/100]   topology_experimental_raft  debug  [ PASS ] topology_experimental_raft.test_tablets.71 71.83s
[72/100]   topology_experimental_raft  debug  [ PASS ] topology_experimental_raft.test_tablets.72 73.19s
[73/100]   topology_experimental_raft  debug  [ PASS ] topology_experimental_raft.test_tablets.73 70.74s
[74/100]   topology_experimental_raft  debug  [ PASS ] topology_experimental_raft.test_tablets.74 72.42s
[75/100]   topology_experimental_raft  debug  [ PASS ] topology_experimental_raft.test_tablets.75 75.02s
[76/100]   topology_experimental_raft  debug  [ PASS ] topology_experimental_raft.test_tablets.76 73.71s
[77/100]   topology_experimental_raft  debug  [ PASS ] topology_experimental_raft.test_tablets.77 74.68s
[78/100]   topology_experimental_raft  debug  [ PASS ] topology_experimental_raft.test_tablets.78 74.18s
[79/100]   topology_experimental_raft  debug  [ PASS ] topology_experimental_raft.test_tablets.79 71.93s
[80/100]   topology_experimental_raft  debug  [ PASS ] topology_experimental_raft.test_tablets.80 74.48s
[81/100]   topology_experimental_raft  debug  [ PASS ] topology_experimental_raft.test_tablets.81 71.59s
[82/100]   topology_experimental_raft  debug  [ PASS ] topology_experimental_raft.test_tablets.82 73.84s
[83/100]   topology_experimental_raft  debug  [ PASS ] topology_experimental_raft.test_tablets.83 73.05s
[84/100]   topology_experimental_raft  debug  [ PASS ] topology_experimental_raft.test_tablets.84 72.39s
[85/100]   topology_experimental_raft  debug  [ PASS ] topology_experimental_raft.test_tablets.85 72.72s
[86/100]   topology_experimental_raft  debug  [ PASS ] topology_experimental_raft.test_tablets.86 72.33s
[87/100]   topology_experimental_raft  debug  [ PASS ] topology_experimental_raft.test_tablets.87 88.45s
[88/100]   topology_experimental_raft  debug  [ PASS ] topology_experimental_raft.test_tablets.88 73.24s
[89/100]   topology_experimental_raft  debug  [ PASS ] topology_experimental_raft.test_tablets.89 71.75s
[90/100]   topology_experimental_raft  debug  [ PASS ] topology_experimental_raft.test_tablets.90 72.08s
[91/100]   topology_experimental_raft  debug  [ PASS ] topology_experimental_raft.test_tablets.92 72.37s
[92/100]   topology_experimental_raft  debug  [ PASS ] topology_experimental_raft.test_tablets.91 87.16s
[93/100]   topology_experimental_raft  debug  [ PASS ] topology_experimental_raft.test_tablets.93 72.94s
[94/100]   topology_experimental_raft  debug  [ PASS ] topology_experimental_raft.test_tablets.94 71.63s
[95/100]   topology_experimental_raft  debug  [ PASS ] topology_experimental_raft.test_tablets.95 74.10s
[96/100]   topology_experimental_raft  debug  [ PASS ] topology_experimental_raft.test_tablets.96 72.98s
[97/100]   topology_experimental_raft  debug  [ PASS ] topology_experimental_raft.test_tablets.97 69.28s
[98/100]   topology_experimental_raft  debug  [ PASS ] topology_experimental_raft.test_tablets.98 89.23s
[99/100]   topology_experimental_raft  debug  [ PASS ] topology_experimental_raft.test_tablets.100 71.24s
[100/100]  topology_experimental_raft  debug  [ PASS ] topology_experimental_raft.test_tablets.99 73.89s
------------------------------------------------------------------------------
CPU utilization: 13.1%
```

File: test/topology_experimental_raft/test_tablets.py [hunk 1]
```diff
@@ -419,7 +419,7 @@ async def check():
 
     await cql.run_async("DROP KEYSPACE test;")
 
-@pytest.mark.skip(reason="failing a lot, see https://github.com/scylladb/scylladb/issues/16859")
+#@pytest.mark.skip(reason="failing a lot, see https://github.com/scylladb/scylladb/issues/16859")
```
Comment (denesb) [inline test/topology_experimental_raft/test_tablets.py:422]: Please remove, not just comment out.

**Comment by denesb** [comment]
> This PR fixes test_tablet_missing_data_repair and enable the test again.

Please also mention briefly in the cover letter, the reason the test was failing.

File: test/topology_experimental_raft/test_tablets.py (hunk 1 above)
Comment (asias) [reply test/topology_experimental_raft/test_tablets.py]: Done.

**Comment by asias** [comment]
> > This PR fixes test_tablet_missing_data_repair and enable the test again.
> 
> Please also mention briefly in the cover letter, the reason the test was failing.

Updated.

**Review by denesb — APPROVED** [review]

**Comment by asias** [comment]
Triggering one more CI to have more confidence.

**Comment by denesb** [comment]
> Triggering one more CI to have more confidence.

I already queued it.

**Comment by asias** [comment]
> > Triggering one more CI to have more confidence.
> 
> I already queued it.

OK. We have 200 runs passed locally + 2 CI passes already. Let's see if the CI can pass consistently.

**Pull request closed without merging**